lnrpc/walletrpc/walletkit.proto

```diff
@@ -995,6 +995,22 @@ message LabelTransactionRequest {
 message LabelTransactionResponse {
 }
 
+// The possible change address types for default/imported accounts. By default,
```

Comment:
Remove the "/imported" here, as those can't specify a change key scope.

Reply:
Actually, it's confusing indeed.

What I meant by "imported" account is the:

https://github.com/lightningnetwork/lnd/blob/e520d5e79c07eb4c203228334fb2f04d399f026a/lnwallet/btcwallet/psbt.go#L69
It is a namespace and it behaves like "default" account

The "imported" account can have various key scopes (compared to "custom accounts")

If you think that it is too confusing, I can indeed remove "imported"

Reply:
Ah, yes, the specific `"imported"` account name... But I think that's only used for single keys (e.g. from `ImportPubKey`), which is a detail the RPC user might not know. So maybe we could use the phrasing "default accounts or single imported public keys"?

Reply:
Looks good to me. I will change